packages/jest/test/printer.test.js

```diff
@@ -149,18 +149,6 @@ header .emotion-0 {
   })
 })
 
-test('throws nice error for invalid css', () => {
-  const tree = renderer.create(<div css={css`jnnjvh@'jevhevhb`} />).toJSON()
```

Comment:
stylis produces an empty rule for this, so this invalid stuff gets lost and we do not have an easy way to throw a nice error for this

Reply:
This error existed because the css parser we use in @emotion/jest threw a useless error on invalid rules so as long as there's no useless errors, I think it's fine